types/migrations/20220714123910_edited_collections_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
+create type chapter_t as enum ('intro', 'body');
```

Comment:
For a `SQL` enum, we need to ensure it works with a corresponding Rust type. Generally that means UpperCamelCase for enum variants. In short, let's do `Intro` and `Body` for these variants too.